### ISSUE TITLE
VO focus jumps after interacting with an object/control

### ISSUE THREAD

**Issue by Nancy-Salpepi** [phetsims/gravity-force-lab-basics#305]
**Test device**
iPad Air 2

**Operating System**
14.7.1

**Browser**
safari

**Problem description**
https://github.com/phetsims/qa/issues/702

After interacting with an object/control such as a spinner or a checkbox, on the next swipe of my finger, VO sometimes jumps to a completely different area. Some examples include to the sim description or to the preferences button. This does not always occur and where VO focus moves to changes. 

For example, in the attached movie, after the distance checkbox is unchecked, I swiped left. It should have moved to the forces values checkbox, but instead it moved to Sphere Positions. 

**Steps to reproduce**
Since this issue occurs sporadically, these steps with not work all of the time:
1. open sim
2. turn on VO by pressing the home button 3 times
3. tab to the distance check box
4. double click to uncheck the checkbox. 
5. swipe left to move to the Force Values checkbox.

**Visuals**

https://user-images.githubusercontent.com/87318828/132424112-698d79ae-6656-4993-bfc3-db08d02d1650.mov

**Comment by Nancy-Salpepi** [comment]
I have also seen this occur after a spinner is manipulated. 
In general, VO works correctly for several minutes and then one of these inconsistencies will occur.

**Comment by jessegreenberg** [comment]
Hmm, I am not sure why this is happening. I noticed in the video recording that at around 5 seconds there is a single tap and VO makes a low beep sound. It could be related, I will try it on my device.

**Comment by Nancy-Salpepi** [comment]
Yes @jessegreenberg please do. Sometimes I'm not sure if it is because I'm just new at using VO with the iPad or if it is a real issue.

**Comment by jessegreenberg** [comment]
Understood! What you have shown is not what I would expect so Im glad you made an issue.

**Comment by Nancy-Salpepi** [comment]
Here is another example (I didn't do any accidental taps in this one).

https://user-images.githubusercontent.com/87318828/132549727-73909b33-a901-4d6d-91a4-5501e9e28f77.mov

**Comment by jessegreenberg** [comment]
Thanks @Nancy-Salpepi - I also hear the same low beep sound in that video recording at the 15 second mark, wondering if it could be related.

**Comment by Nancy-Salpepi** [comment]
Yeah that sound happens when I swipe and VO focus doesn't change. It makes that noise instead.

**Comment by jessegreenberg** [comment]
OK, I played around with the sim on my iPad Air 2 and I think I see what is happening. I don't think this is a bug but just a result of the gesture shown in the video capture in https://github.com/phetsims/gravity-force-lab-basics/issues/305#issuecomment-915394908.

The Preferences button in the navigation bar uses `positionInPDOM: true`, which means if you tap on it VO will move the virtual cursor there. Actually, if you tap anywhere near the button (in the lower left area of the sim) VO will will consider that close enough and move the cursor there.

I think what is happening is that the gestures in https://github.com/phetsims/gravity-force-lab-basics/issues/305#issuecomment-915394908 are not quite fast enough for Apple to consider them swipes and considers them drag and release motions instead.

@Nancy-Salpepi I think if you swipe a little more quickly (almost like a flick) this issue should go away. I found this really nice example video of someone showing an example of the swipe gesture if you are interested, timestamp 4:50 https://youtu.be/PR5fzIeh-fY?t=290

**Comment by Nancy-Salpepi** [comment]
Thanks for the video @jessegreenberg. Things are working much better now. You have been helpful as always. I figured it was me, which is why I was hesitant to open up an issue in the first place. When you swipe up with 2 fingers to start from beginning, does VO start from the beginning of the sim or from the header above the sim? Mine starts at the back arrow. Does that mean I'm not swiping correctly with two fingers either?

**Comment by jessegreenberg** [comment]
Hooray, so glad to hear it @Nancy-Salpepi! Always very helpful to have issues to investigate, I appreciate you finding them and creating them (including this one!).

>When you swipe up with 2 fingers to start from beginning, does VO start from the beginning of the sim or from the header above the sim? Mine starts at the back arrow.

To be honest mine starts at the "Heading level 1" of the simulation, not the back arrow. But I doubt you are doing anything wrong, maybe it is just a setting or difference in iOS version. Since you are generally able to move the cursor to the top it sounds like you are doing it correctly. If you would like to post a capture of the gesture you are making we can take a look, but I think it is OK to close.

@Nancy-Salpepi feel free to close if you agree.

**Comment by Nancy-Salpepi** [comment]
I never would have thought I would fail at swiping! Haha. 
I didn't have any issues when using a shortened flick gesture yesterday and just tried again today..so I do think it is OK to close.
